simulators/gromacs/v2022.2/Dockerfile

```diff
@@ -11,10 +10,56 @@ RUN apt-get update && apt-get install -y \
 COPY ./test_sim.sh /home/test_sim.sh
 RUN chmod +x /home/test_sim.sh
 
-# Start from a clen GROMACS image
-FROM gromacs/gromacs:2022.2
+From ubuntu:22.04 as build
 
-# Set environment variable to allow GPU direct communication
-ENV GMX_ENABLE_DIRECT_GPU_COMM=1
+ENV OMPI_ALLOW_RUN_AS_ROOT=1
+ENV OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
```

Comment:
Not sure if this should exist here, it is not adequate to allow run as root in HPC, since it allows root from inside the container to do whatever it wants. In principle, inside the container users can do `--allow-run-as-root` and hence at least they are more aware of it.

In any case, we should start considering applying `Users` inside the dockerfiles, wdyt @sergiomsantos ?

Reply:
If I don't use this ENV var the frolics test that use mpi fail because we are running them as root.

Reply:
Yes, I know, but the reason is to use `--allow-run-as-root` which implies that you are actually overriding it.